pwnlib/elf/corefile.py

```diff
@@ -535,6 +927,362 @@ def __getattr__(self, attribute):
             if hasattr(self.prstatus.pr_reg, attribute):
                 return getattr(self.prstatus.pr_reg, attribute)
 
-        return super(Core, self).__getattribute__(attribute)
+        return super(Corefile, self).__getattribute__(attribute)
+
+class Core(Corefile):
+    """Alias for :class:`.Corefile`"""
```

Comment:
It seems ugly to have a diffent class for this. Would it be possible to make it actua alias and hack the docs somehow?